be/src/common/config.h

```diff
@@ -938,6 +938,14 @@ CONF_mInt64(l0_max_mem_usage, "67108864");  // 64MB
 CONF_mInt64(l0_snapshot_size, "16777216"); // 16MB
 CONF_mInt64(max_tmp_l1_num, "10");
 CONF_mBool(enable_parallel_get_and_bf, "true");
+CONF_mBool(enable_pindex_filter, "true");
+CONF_mInt32(max_pages_ratio, "20");
```

Comment:
What's usage of this configuration?

Reply:
> What's usage of this configuration?

`enable_pindex_filter` is used to enable or disable bloom filter for pindex.

Reply:
What's the difference between reading by page or by shard? Much performance difference?

Reply:
> What's the difference between reading by page or by shard? Much performance difference?

Read by page can reduce disk IO compared to Read by shard.
We need to read a total shard even if we only need to read one record. Read by page is hard to work without bloom filter because the data distribution is almost random(by hash). So even we have a batch data to check, we maybe need to read multiple shards data and most data is useless because the real data we need is very small. If we use bloom filter to filter some record before read data from disk, we may find the real data we need to read in one shard is very small because some record is filtered before, so we don't need to read total shard data.